.gitignore

```diff
@@ -6,4 +6,4 @@ node_modules
 *.iml
 *.code-workspace
 .DS_Store
-sakila.db
```

Comment:
suggestion: `sakila.db` is created and modified while running the SQLite integration tests. We don't want to check it by accident, so this should be left as-is.